### ISSUE TITLE
en_GB causing error

### ISSUE THREAD

**Issue by gazzer82** [elixir-cldr/cldr#165]
I am trying to set a locale options, as well as the default locale to en_GB, but I get the following error when starting up the Phoenix project.

 ex_cldr version 2.24.2

Here's my config:

defmodule Unipix.Cldr do
  use Cldr,
    locales: ["en_GB"],
    default_locale: "en_GB",
    providers: [Cldr.Number, Money]
end

```
Generating Unipix.Cldr for 3 locales named ["en", "en-GB", "und"] with a default locale named "en_GB"

== Compilation error in file lib/unipix/cldr.ex ==
** (BadMapError) expected a map, got: nil
    (stdlib 3.16.1) :maps.put(:k, :v, nil)
    (stdlib 3.16.1) erl_eval.erl:252: :erl_eval.expr/5
    (stdlib 3.16.1) erl_eval.erl:893: :erl_eval.expr_list/6
    (stdlib 3.16.1) erl_eval.erl:408: :erl_eval.expr/5
    (ex_cldr 2.24.2) /Users/garethjeanne/Development/up/2021/unipix/lib/unipix/cldr.ex:1: Cldr.Backend.Compiler.__before_compile__/1
```

Any pointers on where I'm going wrong here, apologies if it's something obvious!

**Comment by kipcole9** [comment]
I have published [ex_cldr version 2.25.0](https://hex.pm/packages/ex_cldr/2.25.0) which fixes this issues. Please run `mix deps.update ex_cldr` in your project.  The changelog entry reads:

### Bug Fixes

* Fixes configuring locales when the default locale is in posix form (ie like `en_GB`) as apposed to BCP47 form (ie `en-GB`). In fixing this bug, forming the normalised list of configured locales is now also standarised. Thanks to @gazzer82 for the report. Closes #165

### Enhancements

* Implement `Cldr.Gettext.Plural` allowing the creation of [gettext](https://hexdocs.pm/gettext) [plural forms](https://hexdocs.pm/gettext/Gettext.Plural.html#content) modules.

Thanks for the support and collaboration!

**Comment by gazzer82** [comment]
Yup all working fine now, thanks so much for sorting so quickly, Merry Christmas!
